compound-comptroller-bot/src/finding.ts

```diff
@@ -0,0 +1,46 @@
+import { ethers, Finding, FindingSeverity, FindingType } from "forta-agent";
+
+export const createFinding = (log: ethers.utils.LogDescription): Finding => {
+  switch (log.signature) {
+    case "ActionPaused(address,string,bool)":
+      return Finding.from({
```

Comment:
The `Finding`s seem to be missing the `addresses` property with the emitting address as the value..